spring-cloud-config-client/.gitignore

```diff
@@ -0,0 +1 @@
+/.apt_generated/
```

Comment:
I removed it, and added some tests to improve code coverage, and test that the change actually performed as expected.